rtv/config.py

```diff
@@ -1,5 +1,36 @@
 """
 Global configuration settings
 """
-
 unicode = False
+
+default_colors = {
```

Comment:
All of this extra whitespace goes against the standard python convention. See https://www.python.org/dev/peps/pep-0008/#pet-peeves for reference.